packages/react/runtime/src/lynx/performance.ts

```diff
@@ -65,11 +76,18 @@ function markTimingLegacy(key: PerformanceTimingKeys, timingFlag_?: string): voi
   lynx.getNativeApp().markTiming?.(timingFlag!, PerformanceTimingKeys[key]);
 }
 
-function beginPipeline(needTimestamps: boolean, timingFlag?: string): void {
+function beginPipeline(needTimestamps: boolean, pipelineOrigin: string, timingFlag?: string): void {
```

Comment:
Use strict type instead
```suggestion
function beginPipeline(needTimestamps: boolean, pipelineOrigin: PipelineOrigins[keyof PipelineOrigins], timingFlag?: string): void {
```